XenAdmin/Images.cs

```diff
@@ -592,338 +594,345 @@ public static Icons GetIconFor(DockerContainer dockerContainer)
 
         public static class StaticImages
         {
-            // the followings are generated from Resources using:
+            // the following are generated from Resources using:
             // cat Resources.Designer.cs | grep 'internal static System.Drawing.Bitmap' | sed "s/ {//g" | awk -F"Bitmap " '{print"public static Image " $2 " = Properties.Resources." $2 ";"}'

```

Comment:
Please change the `cat` command to `public static Bitmap` also in case they are regenerated based on it.